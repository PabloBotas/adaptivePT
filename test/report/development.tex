\documentclass{article}
\usepackage[a4paper, total={6in, 8in}]{geometry}
\usepackage[utf8]{inputenc}
\usepackage{graphicx}

\usepackage[pdftex,
            pdfauthor={Pablo Botas},
            pdftitle={Development report}]{hyperref}

\title{Development report}
% \date{}
\author{Pablo Botas}

\begin{document}
\maketitle

\section{Ray Tracing}

\begin{enumerate}
    \item A single ray is initialized per spot.
    \item The ray losses energy following the CSDA.
    \item When the ray has zero energy the endpoint is scored.
\end{enumerate}

\begin{figure}[h]
    \includegraphics[width=0.49\textwidth, angle=90]{beam.png}
    \includegraphics[width=0.49\textwidth]{profile.png}
    \caption{Beam with no $\sigma$ and $\epsilon$ in a patient (P15) and ray traced trajectory.}
\end{figure}

\section{Vector field probing}


\end{document}