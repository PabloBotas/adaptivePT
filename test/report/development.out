\BOOKMARK [1][-]{section.1}{Ray Tracing}{}% 1
